roles/openshift_examples/examples-sync.sh

```diff
@@ -9,7 +9,7 @@ XPAAS_VERSION=ose-v1.3.3
 ORIGIN_VERSION=${1:-v1.3}
 EXAMPLES_BASE=$(pwd)/files/examples/${ORIGIN_VERSION}
 find ${EXAMPLES_BASE} -name '*.json' -delete
-find ${EXAMPLES_BASE} -name '*.yaml' -delete
+find ${EXAMPLES_BASE} -name '*.yaml' -delete -exclude registry-console.json
```

Comment:
yuck...  should this be put in a separate directory under `roles/openshift_examples/files`?